apps/web/src/views/Farms/Farms.tsx

```diff
@@ -370,30 +410,93 @@ const Farms: React.FC<React.PropsWithChildren> = ({ children }) => {
+                        {t('Farm Types')}
+                        {farmTypesEnableCount > 0 && `(${farmTypesEnableCount})`}
+                      </Button>
+                    }
+                    isOpen={isOpen}
+                    options={{ placement: 'top' }}
+                  >
+                    <Box width="375px" ref={menuRef}>
+                      <FarmTypesWrapper alignItems="center" p="16px">
+                        <Text fontSize={20} bold color="text" display="inline-block" ml="8px">
+                          {t('Farm Types')}
+                        </Text>
+                      </FarmTypesWrapper>
+                      <Box height="240px" overflowY="auto">
+                        <StyledItemRow alignItems="center" px="16px" py="8px" ml="8px" mt="8px">
+                          <RocketIcon />
+                          <Text fontSize={16} ml="10px" style={{ flex: 1 }} bold>
+                            {t('Booster Available')}
+                          </Text>
+                          <ToggleWrapper>
+                            <Toggle
+                              id="boosted-only-farms"
+                              checked={boostedOnly}
+                              onChange={() => {
+                                setFarmTypesEnableCount((count) => {
+                                  const totalFarmsEnableCount = count + (!boostedOnly ? 1 : -1)
+                                  return totalFarmsEnableCount
+                                })
+
+                                setBoostedOnly((prev) => !prev)
+                              }}
+                              scale="sm"
+                            />
+                          </ToggleWrapper>
+                        </StyledItemRow>
+                        <StyledItemRow alignItems="center" px="16px" py="8px" ml="8px" mt="8px">
+                          <CurrencyIcon />
+                          <Text fontSize={16} ml="10px" style={{ flex: 1 }} bold>
+                            {t('StableSwap')}
+                          </Text>
+                          <ToggleWrapper>
+                            <Toggle
+                              id="stableSwap-only-farms"
+                              checked={stableSwapOnly}
+                              onChange={() => {
+                                setFarmTypesEnableCount((count) => {
+                                  const totalFarmsEnableCount = count + (!stableSwapOnly ? 1 : -1)
+                                  return totalFarmsEnableCount
+                                })
+
+                                setStableSwapOnly((prev) => !prev)
+                              }}
+                              scale="sm"
+                            />
+                          </ToggleWrapper>
+                        </StyledItemRow>
+                      </Box>
+                    </Box>
+                  </InlineMenu>
+                </Box>
+              </Flex>
```

Comment:
For this part could you create a component?

Reply:
done